controllers/chat/group/leaveGroupMember.js

```diff
@@ -0,0 +1,34 @@
+const BetterSocialCore = require('../../../services/bettersocial');
+const {ResponseSuccess} = require('../../../utils/Responses');
+const ErrorResponse = require('../../../utils/response/ErrorResponse');
+const Getstream = require('../../../vendor/getstream');
+
+const leaveGroupMember = async (req, res) => {
+  const {userId} = req;
+  const {channelId} = req.body;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good practice to validate inputs before using them. Although you have Joi validation in place, it would be better to also check if `channelId` is not null or undefined before using it. 

```diff
-  const {channelId} = req.body;
+  const {channelId} = req.body;
+  if (!channelId) {
+    return ErrorResponse.e400(res, 'Channel ID is required');
+  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->